projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -398,6 +398,19 @@ private function parse_menu_item( $title ) {
 			$title = trim( str_replace( $matches[0], '', $title ) );
 		}
 
+		if ( false !== strpos( $title, 'inline-text' ) ) {
+			preg_match( '/<span class="inline-text">([A-Za-z0-9]+)<\/span>/', $title, $matches );
+
+			$text = $matches[1];
+			if ( $text ) {
+				// Keep the text in the item array.
+				$item['inlineText'] = $text;
```

Comment:
Not that important, but can you update the `get_item_schema()` with this new property (the same should be done in client/state/admin-menu/schema.js)? :)

Reply:
Done for this repo!